commands/serve/web/__tests__/connect.test.js

```diff
@@ -460,7 +462,38 @@ describe('connect.js', () => {
           secure: expect.any(Boolean),
           host: expect.any(String),
           port: undefined, // because of providing a link
-          appId: 'SOME_APP_ID/', // since there is not appid in link
+          appId: 'SOME_APP_ID/', // since there is an appid in link
+          prefix: undefined,
+        }),
+        engineUrl: expect.any(String),
+        appUrl: url,
+      });
+    });
+
+    test('should match prefix correctly', () => {
+      url = `/app/SOME_APP_ID/?engine_url=wss://${authConfig.host}/prefix`;
+      const result = parseEngineURL(url);
+      expect(result).toMatchObject({
+        enigma: expect.objectContaining({
+          secure: expect.any(Boolean),
+          host: expect.any(String),
+          port: undefined, // because of providing a link
+          appId: 'SOME_APP_ID/', // since there is an appid in link
+          prefix: 'prefix',
+        }),
+        engineUrl: expect.any(String),
+        appUrl: url,
+      });
+
+      url = `wss://${authConfig.host}/prefix/app/SOME_APP_ID/`;
```

Comment:
testing with
url = `wss://${authConfig.host}/app/SOME_APP_ID/`;
give the questionable prefix `app`

Reply:
you can likely create a virtual proxy named app, so if you input that it should be fine. Depends on the mode. I don't think the /app part is even used here, as that is a different query param.

Reply:
yea I was wrong again, it is indeed used.